pkg/executables/tink.go

```diff
@@ -57,7 +57,9 @@ func (t *Tink) GetHardware(ctx context.Context) ([]*hardware.Hardware, error) {
 			return nil, fmt.Errorf("error unmarshling hardware json: %v", err)
```

Comment:
Is the indentation correct here?

Reply:
I guess yes otherwise l would have got linting errors.